spec/text_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'text'
+
+describe Text do
+  let(:text_send_stub) {
+    stub_request(:post, "https://api.twilio.com/2010-04-01/Accounts/ACeeb78433dcba23619b98cde03e52ad52/Messages.json").
```

Comment:
It should be possible to mock this in a much simpler way - try simplifying the mocking of the Twilio object to make this test easier to read